docsteady/vcd.py

```diff
@@ -553,14 +557,18 @@ def do_req_coverage(ves, ve_coverage):
     """
     Calculate the coverage level of a requirement
     based on the downstram verification elements.
+    :param myvreq: version requirement name
     :param ves:
     :param ve_coverage:
     :return:
     """
     nves = len(ves)
     vecount = Counter()
     for ve in ves:
-        vecount.update([ve_coverage[ve]['coverage']])
+        element = ve_coverage[ve]
+        # there is only one VE per requirement ?
```

Comment:
Is that a question for the person reading the code or is it saying that the value of `cover` indicated where there is one VE per requirement?

Reply:
I guess it was my comment looking at the code - it seems to be implied but its not commented and theoretically may not be true .. I will update